README.md

```diff
@@ -1,8 +1,12 @@
 # operator-controller
-// TODO(user): Add simple overview of use/purpose
+The operator-controller is the central component of Operator Lifecycle Manager (OLM) v1. It extends Kubernetes with an API through which users can install Operators.
```

Comment:
My intent with these two sections in the README was not to fully flesh them out yet but mostly just to get rid of the TODOs :slightly_smiling_face: